.github/workflows/stale.yml

```
@@ -0,0 +1,57 @@
+name: 'Close stale issues and PRs'
+
+on:
+  schedule:
+    # Daily at 1:30 AM UTC
+    - cron: '30 1 * * *'
+
+env:
+  LC_ALL: en_US.UTF-8
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Stale Action"
+        uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+        with:
+          # Apply the `stale` label to an issue with no activity for 90 days.
+          days-before-issue-stale: 90
+          # Close the issue if no activity occurs within 30 days after being
+          # marked as stale.
+          days-before-issue-close: 30
+          stale-issue-label: 'stale'
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not
+            had activity within 90 days. It will be automatically closed if no
+            further activity occurs within 30 days. Leave a comment if
+            you feel this issue should remain open. Thank you!
+          close-issue-message: >
+            This issue has been automatically closed due to inactivity. Please
+            feel free to reopen if you feel it is still relevant. Thank you!
+
+          # Apply the `stale` label to a pull request with no activity for 90 days.
+          days-before-pr-stale: 90
+          # Close the pull request if no activity occurs within 30 days after
+          # being marked as stale.
+          days-before-pr-close: 30
```

I think the good variable names make these comments redundant

```suggestion
          days-before-pr-stale: 90
          days-before-pr-close: 30
```